### ISSUE TITLE
escapeshellarg() necessary for binary path

### ISSUE THREAD

**Issue by leemcd56** [hendrikmaus/drafter-php#10]
I am working on a project that is synced through Dropbox and unfortunately contains spaces and parenthesis in the path. Because of this, on line 252 of `Drafter.php` I had to add `escapeshellarg()` around variable containing the binary path to continue.

**Current:**
```php
$command = $this->binary . ' ' . implode(' ', $options);
```

**Change:**
```php
$command = escapeshellarg($this->binary) . ' ' . implode(' ', $options);
```

**Comment by hendrikmaus** [comment]
Would you create a pull-request for the change? If the tests pass, I do not see a reason not to include it.

**Comment by leemcd56** [comment]
Should be resolved by pull request #11 :)
